internal/cluster/cluster.go

```diff
@@ -211,6 +216,64 @@ func (m *Manager) DeregisterCluster(ctx context.Context, c *common.Cluster) erro
 	return err
 }
 
+func (m *Manager) reportValidationFailedMetrics(ctx context.Context, c *common.Cluster, ocpVersion, emailDomain string) error {
+	log := logutil.FromContext(ctx, m.log)
+	var validationRes validationsStatus
+	if c.ValidationsInfo != "" {
+		if err := json.Unmarshal([]byte(c.ValidationsInfo), &validationRes); err != nil {
+			log.WithError(err).Errorf("Failed to unmarshal validations info from cluster %s", *c.ID)
+			return err
+		}
+	}
+	for _, vRes := range validationRes {
+		for _, v := range vRes {
+			if v.Status == ValidationFailure {
+				m.metricAPI.ClusterValidationFailed(ocpVersion, emailDomain, models.ClusterValidationID(v.ID))
```

Comment:
It existed in host validations metrics and was removed by @gamli75 as part of reducing the metrics quantity that causes some issues in app-are prometheus.

Reply:
Seems strange, so we get random metrics with no mapping to host/cluster?
@gamli75?

Reply:
We can filter by clsuterID using the events in Elastic, regarding the metrics, we only need to know what are the scales - which validations failed more than other and made the user eventually deleted the cluster.

Reply:
But you don't send events for successful / failed validations, you only send event in case validation changed status.
So elastic data will be partial.

Reply:
Correct.
As I said, clusterID was removed from metrics because in overloaded app-are prometheus and @gamli75 has decided to remove it, bringing it back again will contradict the changes we did 2 weeks ago.